articles/reporting/summary.md

```diff
@@ -3,15 +3,15 @@ title: "Summary"
 description: "The Reports tab will load in the summary screen view with a default time range of the last hour For large senders there may be a short delay while the data is being compiled The Calendar drop down provides preset time ranges and an optional custom date time picker You..."
 ---
 
-The Reports tab will load in the summary screen view, with a default time range of the last hour.  For large senders, there may be a short delay while the data is being compiled. The Calendar drop down provides preset time ranges and an optional custom date/time picker. You can also apply additional search criteria like recipient domain, campaign or template name in the search bar in the upper right.
+Summary in the Reports menu will load in the summary screen view, with a default time range of the last hour.  For large senders, there may be a short delay while the data is being compiled. The Calendar drop down provides preset time ranges and an optional custom date/time picker. You can also apply additional search criteria like recipient domain, campaign or template name in the search bar in the upper right.
```

Comment:
I understand the change here, but it reads kind of weird IMO. 
> The Summary button under the Reports menu will load...

( button | tab | nav-item)?

Reply:
Summary item in the reports menu?  its not really a tab.